UltraStar Play/Assets/Scenes/SongSelect/SongRoulette/SongRouletteController.cs

```diff
@@ -232,6 +232,19 @@ public void SelectSong(SongMeta songMeta)
         Selection.Value = new SongSelection(songMeta, songs.IndexOf(songMeta), songs.Count);
     }
 
+    public void SelectSongByIndex(int index, bool wrapAround = true)
+    {
+        if (!wrapAround
+            && (index < 0 || songs.Count <= index))
+        {
+            // Ignore out-of-range index
+            return;
```

Comment:
This should probably be logged so that faulty code can be found and fixed?

Reply:
IMO wrapAround as parameter should ignore out-of-bounds index. For example you might want the previous index and do (index-1). This could be out-of-range but would not be an error from my point of view. It just does nothing if wrapAround is false.

Works as intended.